core/src/main/java/com/linecorp/armeria/common/StringMultimap.java

```diff
@@ -431,6 +464,12 @@ public final boolean containsObject(IN_NAME name, Object value) {
         return contains(name, fromObject(value));
     }
 
+    @Override
+    public final boolean containsBoolean(IN_NAME name, boolean value) {
+        final Boolean v = getBoolean(name);
+        return v != null && v == value;
+    }
+
```

Comment:
```
    @Override
    public final boolean containsInt(IN_NAME name, int value) {
        return contains(name, String.valueOf(value));
    }
```
`containsBoolean` should use `contains()` same with other `contains*`? 🤔
Fix PR: https://github.com/line/armeria/pull/5319